### DISCUSSION TITLE
Problems using browser to evoke webwiew2 app

### DISCUSSION THREAD

**Discussion by sFireFrog** [MicrosoftEdge/WebView2Feedback#4059] [Q&A]
The webview2 is embedded in the app, and it cannot be called in the browser using the windows custom protocol. Is there any good solution?
The browser calls another console program, which then launches the application. no problem.
win32 for webview2 app

**Comment by Danielx64** [comment]
I been able to create a custom schema and get my app to launch with webview2 up and being able to display any page I want

**Comment by sFireFrog** [comment]
> I been able to create a custom schema and get my app to launch with webview2 up and being able to display any page I want

Is it a program launched in the browser using a custom protocol?

**Comment by Danielx64** [comment]
Yep,

I can call ms-mobile-apps://.... in edge and other browsers and it will launch my app. You - need to set this up in the registry as well as your app to make it work

**Comment by Danielx64** [comment]
https://stackoverflow.com/questions/80650/how-do-i-register-a-custom-url-protocol-in-windows

**Comment by sFireFrog** [comment]
> Yep,
> 
> I can call ms-mobile-apps://.... in edge and other browsers and it will launch my app. You - need to set this up in the registry as well as your app to make it work

This step is fine, the problem is that the program cannot be initialized normally
It shows that there is no way to load the class of webwiew2

**Comment by Danielx64** [comment]
Hmm I'm not seeing any issues here but I haven't updated to the latest SDK though.

**Comment by sFireFrog** [comment]
> Hmm I'm not seeing any issues here but I haven't updated to the latest SDK though.

Are you using winform or wpf? Is it because I use win32?

**Comment by Danielx64** [comment]
I'm using wpf, C++ hates me
